docs/writing-docs/introduction.md

```diff
@@ -13,7 +13,7 @@ Storybook gives you tools to expand this basic documentation with prose and layo
   />
 </video>
 
-Out of the box, Storybook ships with [DocsPage](./docs-page.md), a documentation template that lists all the stories for a component and associated metadata. It infers metadata values based on source code, types and JSDoc comments. [Customize](./docs-page.md#replacing-docspage) this page to create a new template if you have specific requirements.
+When you install [Storybook Docs](https://github.com/storybookjs/storybook/blob/master/addons/docs/README.md#installation), you can use [DocsPage](./docs-page.md), a documentation template that lists all the stories for a component and associated metadata. It infers metadata values based on source code, types and JSDoc comments. [Customize](./docs-page.md#replacing-docspage) this page to create a new template if you have specific requirements.
```

Comment:
@whoisryosuke instead trying to "tip the scale" (pardon the bad pun) one way or another, we have some room for improvement here and we probably go with:

" If you're including Storybook in your project for the first time, we provide you with [DocsPage](./docs-page.md), a documentation template that lists all the stories for a component and associated metadata. It infers metadata values based on source code, types and JSDoc comments. If you need, you can [customize](./docs-page.md#replacing-docspage) this page to create your own custom template. 

If you're already using Storybook and you're updating to the latest release, we recommend that you install [@storybook/addon-essentials](https://www.npmjs.com/package/@storybook/addon-essentials), to include this and other great features into your project."

Something along these lines. 

Let me know if you're ok with it

Reply:
I think the language is still a little misleading, it should specify how you're running the automatic Storybook install (`init`) somehow vs manual/upgrade. 

```
If you're installing Storybook for the first time with the CLI, we provide you...
If you use the Storybook CLI to install your project, we provide you...
```

Not exactly that maybe? But some reference to the `init` process.

I like the caveat at the end, it helps catch the user's eye with **"updating"** in the next section, so if that's my route, my eyes fall there. Maybe a bold could help draw the eye more 👀

Reply:
@whoisryosuke like i've mentioned we have room for improvement here. 

We probably could go with:

```md
If you're including Storybook in your project for the [first time](../get-started/install.md), we provide you with [DocsPage](./docs-page.md), a documentation template that lists all the stories for a component and associated metadata. It infers metadata values based on source code, types and JSDoc comments. If you need, you can customize this page to create your own custom template.

If you're already using Storybook and you're **updating** to the latest release, we recommend that you install [@storybook/addon-essentials](https://www.npmjs.com/package/@storybook/addon-essentials), to include this and other great features into your project.
```
Beforehand sorry for the shift to markdown, but it's for a bit more clarity. 

Adding mentions to the CLI and init process here would throw people a bit off. More specifically newcomers to Storybook, as they would have to do a bit of search on what is the cli and how to use it. By making the subtle change and include the link to the setup page we can keep the page simple, lean and on point. 

Adding a bold is for the update works for me. I have no issues with it.

Let me know if you're ok with it.

Reply:
No worries, MD works. Looks good.